### DISCUSSION TITLE
cant update adguard

### DISCUSSION THREAD

**Discussion by nmbrg** [tteck/Proxmox#2585] [Help Needed] [answered]
im getting the following error message when i try to update adguard with the 'update' command:

`[ERROR] in line 63: exit code 0: while executing command wget -qL https://static.adguard.com/adguardhome/release/AdGuardHome_linux_amd64.tar.gz`

any suggestions on why this is happening?

**Comment by OddSquirrel** [comment, marked as answer]
Granted, I've heard that before, but if you're referring to AdGuard Home, it can be easily updated right from the GUI.

**Reply by Narkowii** [reply]
+1, i always update Adguard by the GUI

**Comment by nmbrg** [comment]
> Granted, I've heard that before, but if you're referring to AdGuard Home, it can be easily updated right from the GUI.


Yes ofc, but since the recommended way (the way described at https://tteck.github.io/Proxmox/) is to use the 'update' command, i thought it would be a good idea to have it actually work. and since it isnt, i thought id mention it here (as i prefer to do all my updating thru the console instead of going thru a ton of web-ui's for all the services.

**Reply by AlexGeek77** [reply]
'update' also runs the apt upgrade process which helps keep your underlying container up to date. I would still run that and update through the web.

**Reply by nmbrg** [reply]
Well, kinda hard to run something that isn’t working. Have you read my first post? ;)

**Reply by tteck** [reply]
It works, 
```
    ___       __                           __
   /   | ____/ /___ ___  ______ __________/ /
  / /| |/ __  / __  / / / / __  / ___/ __  / 
 / ___ / /_/ / /_/ / /_/ / /_/ / /  / /_/ /  
/_/  |_\__,_/\__, /\__,_/\__,_/_/   \__,_/   
            /____/                           
 
 ✓ Stopped AdguardHome
 ✓ Updated AdguardHome
 ✓ Started AdguardHome
 ✓ Cleaned
 ✓ Updated Successfully
root@adguard:~# 
```

**Reply by tteck** [reply]
But should be used only when the GUI states to manually Update AdGuard Home

**Reply by nmbrg** [reply]
Well it is in fact not working for me. Wouldn’t be here if it was.

**Reply by tteck** [reply]
OK then, the issue is on your end, there is no issue with the script/code.

**Reply by nmbrg** [reply]
Ok, guess I’ll have to use another way to run adguard home then. All I’ve done is run the script. and have no issues updating adguard home on docker.

**Reply by ktfcaptain** [reply]
I had this issue years ago, maybe, and found the container ran out of space.  Increased the storage and rebooted, updated fine.  I'm pretty sure it was downloading the update but failing then not deleting the files, hence getting full, I remember finding all of the files.

I can't remember why, but remember posting here and getting the same kind of responses here saying user error and shrugged it off.  Now I have it running on two hosts and syncing from a third host, not important, but this just happened again on Proxmox, while the instance running on my Synology nas updated without issue.  I again increased the Proxmox container storage, but this time did not find any leftover files from updates needing deleted so I have no idea how it happened again.  

I'm back up but just wanted to share because this drove me crazy for a while when it first happened.
